tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -1965,11 +1964,11 @@ Duration <DURATION>"
 `;
 
 exports[`void_in_expression_position.w 1`] = `
-"error: Property access unsupported on type \\"void\\"
-  --> ../../../examples/tests/invalid/void_in_expression_position.w:1:1
+"error: Property not found
+  --> ../../../examples/tests/invalid/void_in_expression_position.w:1:12
   |
 1 | log(\\"hey\\").get(\\"x\\");
-  | ^^^^^^^^^^ Property access unsupported on type \\"void\\"
+  |            ^^^ Property not found
```

Comment:
This diagnostic is accurate but misleading. The original diagnostic does need some work though so it's probably fine to try this new one out.

Reply:
Yes, that was my thought :-)